domain/machine/state/machine_cloud_instance.go

```diff
@@ -205,19 +205,19 @@ WHERE m.name = $machineName.name;
 
 	var instanceId string
 	err = db.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
-		result := instanceID{}
+		var result instanceID
 		err := tx.Query(ctx, queryStmt, machineIDParam).Get(&result)
 		if err != nil {
+			if errors.Is(err, sqlair.ErrNoRows) {
+				return errors.Annotatef(machineerrors.NotProvisioned, "machine: %q", mName)
+			}
```

Comment:
I'm not going to die on re-writing if statements. If people want to align that, we should bring this up on a cross-team.

Reply:
Fair enough.
However it may worth a discussion with @hpidcock, we agreed on not nesting stuff, but he prefers to keep the `else` (and i understand the point over avoiding someone mess up the code by adding instruction between two error case. My opinion is far less backed, since my only point is "doesn't feels go idiomatic, and default linter tend to flag it" :goat: )